### PR TITLE
Support for iOS platform

### PR DESCRIPTION
**Pull request by miguelangel-dev**
* Create iOS projects
* Support for compile `Sources` files added in Swift Playgrounds
* Support for `Podfiles` with platform `osx` or `ios`

### CODE REVIEW

File: bin/nef-playground [hunk 1]
```diff
@@ -70,14 +73,6 @@ setDependency() {
           target=`cat "$podfilePath" | grep "target"`
           sed -i '' s/"$target"/"target '$projectName' do"/g "$podfilePath"
 
-          platforms=()
```
Comment (miguelangel-dev) [inline bin/nef-playground:73]: before iOS platform support - we force to use `osx`

File: bin/nef-playground [hunk 2]
```diff
@@ -93,6 +88,47 @@ setDependency() {
   fi
 }
 
+##
+#   setPlaygroundPlatform(String projectFolder, String projectName, String podfilePath)
+#   - Parameter `projectFolder`: path to the project folder.
+#   - Parameter `projectName`: the project name.
+#   - Parameter `podfile`
+##
+setPlaygroundPlatform() {
```
Comment (miguelangel-dev) [inline bin/nef-playground:97]: now we have 2 templates: one of them for iOS platform, another for macOS platform

File: bin/nefc [hunk 3]
```diff
@@ -100,10 +100,11 @@ buildProject() {
         # build
         workspace=$(workspaceForProjectPath "$1" "$project")
         workspaceName=$(echo "$workspace" | cut -d'/' -f 2)
+        sdk=$(sdkForWorkspaceAndScheme "$workspace" "$schemeName")
         log="$logPath/$workspaceName.log"
 
         echo -ne "${reset}Building ${green}$workspaceName${normal} ($schemeName) ..."
-        xcodebuild -workspace "$workspace" -scheme "$schemeName" -derivedDataPath "$DERIVED_DATA_DIR" -configuration Debug 1> "$log" 2>&1
+        xcodebuild -workspace "$workspace" -sdk "$sdk" -scheme "$schemeName" -derivedDataPath "$DERIVED_DATA_DIR" -configuration Debug 1> "$log" 2>&1
```
Comment (miguelangel-dev) [inline bin/nefc:107]: we need specify in which SKD we are compiling

File: bin/nefc [hunk 4]
```diff
@@ -283,7 +321,7 @@ makeHeaders() {
     local playground=$2 # parameter `playgroundName`
     local output="nef/build/$playground.swift"
 
-    libs=(`xcrun swiftc -emit-imported-modules "$content" | sort -r`)
+    libs=(`xcrun swiftc -emit-imported-modules "$content" | sort -r | grep -vE PlaygroundSupport`)
```
Comment (miguelangel-dev) [inline bin/nefc:324]: ignore `PlaygroundSupport` - we include a `NefPlaygroundSupport` to keep compatibility between `nef`- `osx` and `iOS`

File: lib/Markup/Parser/MarkupLexical.swift [hunk 5]
```diff
@@ -1,3 +1,5 @@
+//  Copyright © 2019 The nef Authors.
```
Comment (miguelangel-dev) [inline lib/Markup/Parser/MarkupLexical.swift:1]: from here -----> in a different PR

**Review by truizlop — APPROVED** [review]

**Pull request merged**